contents/docs/data/events.mdx

```diff
@@ -114,6 +114,64 @@ Most users of PostHog will want to combine their back-end data, such as user inf
 -   JS [snippet](/docs/integrate)
 -   Client- and server-side [libraries](/docs/integrate)
 
+## Events with person processing
+
+By default all events create or update an associated [person profile](/docs/data/persons). This allows for the following features:
+
+- Filtering on person properties
+- Merging identities across distinct_ids (ie merging anonymous users with your logged in user)
+- Cohort creation
+- Feature flag targeting
+- Tracking initial UTM tags and referrers over time
+
+However, there are many usecases that don't require this person processing. Because events that don't do person processing are cheaper for us to ingest, the pricing for these events is lower.
+
+Some examples of usecases that often don't require user profiles
+- Tracking a marketing website
+- B2C apps where users don't sign up or log in
+- High volume API server events
+
+## PostHog-js
+
+You can control whether events from PostHog-js have person processing enabled with the `person_processing` config setting. There are three options:
+
+- `person_processing: 'always'` _(default)_ - we will process persons data for all events
```

Comment:
Nit: most of our other configs are verbs, eg `enable_recording_console_log` and `capture_pageview`. I wonder if this should be a verb, ie `process_persons`: `always/never/identified_only`